.github/workflows/functional.yml

```diff
@@ -0,0 +1,76 @@
+name: Functional Tests
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+env:
+   TEST_IMAGE_NAME: fiware/orion-ci:rpm7
+
+jobs:
+  functional-1:
+    runs-on: ubuntu-18.04
+
+    services:
+      mongodb:
+        image: mongo:4.0
```

Comment:
Taking into account [this PR that will be merged soon](https://github.com/telefonicaid/fiware-orion/pull/3732) I'd suggest to use 4.2.

Reply:
Ok. Let's keep 4.0 by the moment.

NTC

Reply:
I've set it back to 4.0 in c32fa2e